spec/test-utils/test-data/generate-test-data.py

```diff
@@ -167,6 +168,20 @@ def main() -> None:
 
 /** Signed backup data, suitable for return from `GET /_matrix/client/v3/room_keys/keys/{{roomId}}/{{sessionId}}` */
 export const SIGNED_BACKUP_DATA: KeyBackupInfo = { json.dumps(backup_data, indent=4) };
+
+export const MEGOLM_SESSION_DATA_ARRAY: IMegolmSessionData[] = [
+    { 
+        json.dumps(build_exported_megolm_key(), indent=8)
+    },
+    { 
+        json.dumps(build_exported_megolm_key(), indent=8)
+    },
+];
+
+/** An exported megolm session */
+export const MEGOLM_SESSION_DATA: IMegolmSessionData = { 
+        json.dumps(build_exported_megolm_key(), indent=4)
+};
```

Comment:
do we really need this as well as MEGOLM_SESSION_DATA_ARRAY? can we just use `MEGOLM_SESSION_DATA_ARRAY[0]` ?